dist/s_clang_scan

```diff
@@ -1,11 +1,9 @@
 #!/bin/bash
 
 t_out=__wt.$$.out
-t_err=__wt.$$.err
```

Comment:
We are no longer distinguishing between the stdout and stderr, everything goes to the same file.